blog/content/post/2025-10-30-gigabyte-mz33-ar1-acpi-and-bugfixes.md

```diff
@@ -0,0 +1,252 @@
+- [https://review.coreboot.org/c/coreboot/+/89111](https://review.coreboot.org/c/coreboot/+/89111)
+- [https://review.coreboot.org/c/coreboot/+/89203](https://review.coreboot.org/c/coreboot/+/89203)
+- [https://review.coreboot.org/c/coreboot/+/89204](https://review.coreboot.org/c/coreboot/+/89204)
+- [https://review.coreboot.org/c/coreboot/+/89205](https://review.coreboot.org/c/coreboot/+/89205)
+- [https://review.coreboot.org/c/coreboot/+/89206](https://review.coreboot.org/c/coreboot/+/89206)
+- [https://review.coreboot.org/c/coreboot/+/89207](https://review.coreboot.org/c/coreboot/+/89207)
+- [https://review.coreboot.org/c/coreboot/+/89208](https://review.coreboot.org/c/coreboot/+/89208)
+- [https://review.coreboot.org/c/coreboot/+/89209](https://review.coreboot.org/c/coreboot/+/89209)
+- [https://review.coreboot.org/c/coreboot/+/89475](https://review.coreboot.org/c/coreboot/+/89475)
+- [https://review.coreboot.org/c/coreboot/+/89479](https://review.coreboot.org/c/coreboot/+/89479)
+- [https://review.coreboot.org/c/coreboot/+/89480](https://review.coreboot.org/c/coreboot/+/89480)
+- [https://review.coreboot.org/c/coreboot/+/89488](https://review.coreboot.org/c/coreboot/+/89488)
+- [https://review.coreboot.org/c/coreboot/+/89489](https://review.coreboot.org/c/coreboot/+/89489)
+- [https://review.coreboot.org/c/coreboot/+/89490](https://review.coreboot.org/c/coreboot/+/89490)
+- [https://review.coreboot.org/c/coreboot/+/89792](https://review.coreboot.org/c/coreboot/+/89792)
+- [https://review.coreboot.org/c/coreboot/+/89793](https://review.coreboot.org/c/coreboot/+/89793)
+- [https://review.coreboot.org/c/coreboot/+/89794](https://review.coreboot.org/c/coreboot/+/89794)
+- [https://review.coreboot.org/c/coreboot/+/89807](https://review.coreboot.org/c/coreboot/+/89807)
+- [https://review.coreboot.org/c/coreboot/+/89808](https://review.coreboot.org/c/coreboot/+/89808)
+- [https://review.coreboot.org/c/coreboot/+/89822](https://review.coreboot.org/c/coreboot/+/89822)
+- [https://review.coreboot.org/c/coreboot/+/89823](https://review.coreboot.org/c/coreboot/+/89823)
+- [https://review.coreboot.org/c/coreboot/+/89824](https://review.coreboot.org/c/coreboot/+/89824)
+- [https://review.coreboot.org/c/coreboot/+/89825](https://review.coreboot.org/c/coreboot/+/89825)
+
+We will not go through all of the separately. If you are curious what has been
+changed and why, I encourage you to look into the links and read commit
+messages. The above list of patches fulfills the requirements of the
+milestone:
+
+- Task 5. Platform-feature enablement - Milestone a. Turin-specific ACPI
+  tables
+
+Of course these changes alone are not sufficient to boot the OS reliably. We
+also developed a lot of bugfixes, which we will dive into now.
+
+## Bug fixes
+
+Every plan, even the best plan, will not predict very minor details or issue
+that can pop-up unexpectedly. For that reasons we have defined a separate task
```

Comment:
I agree with you, but we could frame that more in business terms. With every port and every experience, we're getting better. There are always unknown unknowns; the experience is to leave enough buffer and margin for those circumstances.